languages.json

```diff
@@ -1,12 +1,22 @@
 {
   "js": {
     "extension": "js",
+    "outputExtensions": ["js"],
     "image": "node:12-alpine",
-    "cmd": "node"
+    "runCmd": "node"
   },
   "py": {
     "extension": "py",
+    "outputExtensions": ["py"],
     "image": "python:3.7-slim",
-    "cmd": "python"
+    "runCmd": "python"
+  },
+  "ts": {
+    "extension": "ts",
```

Comment:
let's change extension to language or id so it's not confused with outputExtensions